.clang-format

```diff
@@ -79,7 +81,9 @@ SpacesInContainerLiterals: true
 SpacesInCStyleCastParentheses: false
 SpacesInParentheses: false
 SpacesInSquareBrackets: false
-Standard:        Cpp11
```

Comment:
`Cpp11` is deprecated: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#standard